pkg/resource/defaultclass.go

```diff
@@ -96,36 +101,51 @@ func (r *DefaultClassReconciler) Reconcile(req reconcile.Request) (reconcile.Res
 		return reconcile.Result{}, errors.Wrap(IgnoreNotFound(err), errGetClaim)
 	}
 
-	// Get resource classes with claim kind as default
-	classes := &corev1alpha1.ResourceClassList{}
-	if err := r.client.List(ctx, r.options, classes); err != nil {
-		// If this is the first time we encounter no defaults we'll be
+	// Get policies for claim kind in claim's namespace
+	policies := &unstructured.UnstructuredList{}
+	policyList := r.newPolicyList()
```

Comment:
Nitpick: I'd prefer to remove this variable since it's only used once and instead write:

```go
policies.SetGroupVersionKind(r.newPolicyList().GetObjectKind().GroupVersionKind())
```